packages/ui-react/src/components/Popover/Popover.tsx

```diff
@@ -10,11 +10,12 @@ type Props = {
   children: ReactNode;
   isOpen: boolean;
   onClose: () => void;
+  'aria-expanded'?: AriaAttributes['aria-expanded'];
 };
 
-const Popover: React.FC<Props> = ({ children, isOpen, onClose }: Props) => {
+const Popover: React.FC<Props> = ({ children, isOpen, onClose, ...rest }: Props) => {
   return (
-    <Slide open={isOpen} duration={250} direction="right" style={{ position: 'relative' }}>
+    <Slide open={isOpen} duration={250} direction="right" style={{ position: 'relative' }} {...rest}>
```

Comment:
I don't think that the aria attributes are drilled further. Perhaps we have to update the `Slide` and the `Animation` to spread the rest value as well.
Additionally, maybe we can restrict the props more by making them aria attributes only, by defining the props as such:
```typescript
type Props = AriaAttributes & {
  children: ReactNode;
  isOpen: boolean;
  onClose: () => void;
};

const Popover: React.FC<Props> = ({ children, isOpen, onClose, ...ariaAttributes }: Props) => {};
```
```typescript
<Slide open={isOpen} duration={250} direction="right" style={{ position: 'relative' }} {...ariaAttributes}>
```

Reply:
Nice idea, Roy! The props doesn't need to drilled down further, but using `...ariaAttributes` will make the intention more specific (for this specific component), therefore I really like this.

Reply:
You're seeing there's no need to drill further, but currently `<Slide />` doesn't do anything with these (aria) props.

Reply:
Oops. You're right 😅 While I agree with your approach, I realise that `aria-expanded` should not be applied to this component (the destination).  This is something I overlooked. `aria-expanded` is for the button itself (this is already the case).

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded

So no changes in this component are needed.

Reply:
- [x] applied @royschut suggestion to improve code readability

Reply:
Thanks Rachid, but the issue remains that the props aren't used anywhere. So since @langemike points out this prop isn't needed on the element at all, I suggest removing the aria props from both Popover and Slide completely.